src/core/sys/linux/unistd.d

```diff
@@ -18,3 +19,730 @@ enum {
 
 /// Prompt for a password without echoing it.
 char* getpass(const(char)* prompt);
+
+// linux system call number from Linux's asm/unistd.h
+version(X86)
```

Comment:
Wow, this corresponds to 30x file size :-). Could we add this as a another module and publicly import it here?

Reply:
Which file do you suggest?
I put them in this file, because that's where they are defined on Linux.

Reply:
I'm not sure, perhaps `core.sys.linux.syscalls`. I don't have a good answer, and maybe they could go here, indeed.